### DISCUSSION TITLE
Release 3.2.5

### DISCUSSION THREAD

**Discussion by pschichtel** [pschichtel/JavaCAN#51] [Announcements]
The release contains just dependency (most importantly slf4j) upgrades and minor code cleanups.

<hr /><em>This discussion was created from the release <a href='https://github.com/pschichtel/JavaCAN/releases/tag/javacan-3.2.5'>Release 3.2.5</a>.</em>

**Comment by JohnLussmyer** [comment]
I'm trying to figure out if this package will work for me.
I have a Java app running on a Pi.  I need to add J1939 Canbus support.
s = socket(PF_CAN, SOCK_DGRAM, CAN_J1939);
That is the kind of socket I need to create and work with - but looking through the code here, I'm not sure if I'll be able to create it and use it.
Is it possible?  (yes, I'm a very experienced Java programmer - but NOT with Linux system calls.)

**Reply by pschichtel** [reply]
> socket(PF_CAN, SOCK_DGRAM, CAN_J1939);

the `CAN_J1939` protocol is not supported yet. If you can provide me with some reference code on how it's being used I can look into adding it.

**Reply by pschichtel** [reply]
I created #54 to track this
